build.gradle

```diff
@@ -20,7 +20,7 @@ import java.time.format.DateTimeFormatter
 
 plugins {
   id "base"
-  id "com.palantir.consistent-versions" version "1.14.0"
```

Comment:
There is a patch upgrading this plugin (and all of gradle infrastructure) here - https://github.com/apache/solr/pull/275. It'll conflict with your PR, once applied. I'll cherry pick relevant areas if you don't want to spend any more time on it.

Reply:
Ho, I missed this effort upgrading to 7.x, good stuff ! I can wait for your branch to be merged and I'll fix my branch

Reply:
I'm planning to merge tomorrow, sorry for not mentioning this earlier.